File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/LinkEMRPatientInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    public class LinkEMRPatientInfo
    {
        private String _patientId;
        private String _bunho;

        public String PatientId
        {
            get { return this._patientId; }
            set { this._patientId = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public LinkEMRPatientInfo() { }

        public LinkEMRPatientInfo(String patientId, String bunho)
        {
            this._patientId = patientId;
            this._bunho = bunho;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/LinkEMRPatientInfo.java
package nta.med.data.model.ihis.nuro;

import java.math.BigInteger;

public class LinkEMRPatientInfo {
	private BigInteger patientId;
	private String bunhoLink;

	public LinkEMRPatientInfo(BigInteger patientId, String bunhoLink) {
		super();
		this.patientId = patientId;
		this.bunhoLink = bunhoLink;
	}

	public BigInteger getPatientId() {
		return patientId;
	}

	public void setPatientId(BigInteger patientId) {
		this.patientId = patientId;
	}

	public String getBunhoLink() {
		return bunhoLink;
	}

	public void setBunho(String bunhoLink) {
		this.bunhoLink = bunhoLink;
	}
}
